docusaurus.config.js

```diff
@@ -15,11 +15,16 @@ const getDocsDirectoriesForSearchPaths = (rootPath) =>
     .filter((dir) => dir.isDirectory())
     .map((dir) => dir.name);
 
+const isEnvProduction = process.env.VERCEL_ENV === "production";
+const deployUrl = process.env.VERCEL_URL
+  ? `https://${process.env.VERCEL_URL}`
+  : "http://localhost:3000";
```

Comment:
We use 3000 for webchat, so it'll be good to use a different port for convenience.